howl/client/howl_client.py

```diff
@@ -0,0 +1,102 @@
+from howl.context import InferenceContext
+import logging
+import time
+from typing import Callable
+
+import numpy as np
+import pyaudio
+import torch
+from howl.model.inference import InferenceEngine
+
+
+class HowlClient:
+    def __init__(self,
+                 engine: InferenceEngine = None,
+                 context: InferenceContext = None,
+                 device: int = -1,
```

Comment:
what is device for?

Reply:
can't this just be a string?

Reply:
Added a comment "Device for CPU/GPU support. A value of -1 defaults to CPU, positives values will run the model on the associated CUDA device ID.". Would it be fine to ignore multi GPU support for now? Eventually we could change it to pass an array of multiple devices

Reply:
yea we can ignore multi gpu for now